framework/Models.py

```diff
@@ -2151,17 +2390,50 @@ def getAdditionalInputEdits(self,inputInfo):
     for modelIn in self.modelsDictionary.keys():
       self.modelsDictionary[modelIn]['Instance'].getAdditionalInputEdits(inputInfo)
 
-  def run(self,Input,jobHandler):
+  def evaluateSample(self, myInput, samplerType, kwargs):
     """
-      Method to run the essembled model
-      @ In, Input, object, object contained the data to process. (inputToInternal output)
-      @ In, jobHandler, JobHandler instance, the global job handler instance
-      @ Out, None
+        This will evaluate an individual sample on this model. Note, parameters
+        are needed by createNewInput and thus descriptions are copied from there.
+        @ In, myInput, list, the inputs (list) to start from to generate the new one
+        @ In, samplerType, string, is the type of sampler that is calling to generate a new input
+        @ In, **kwargs, dict,  is a dictionary that contains the information coming from the sampler,
+           a mandatory key is the sampledVars'that contains a dictionary {'name variable':value}
+        @ Out, returnValue, dict, This holds the output information of the evaluated sample.
+    """
+    jobHandler = kwargs.pop('jobHandler')
+    Input = self.createNewInput(myInput[0], samplerType, **kwargs)
+
+    ## Unpack the specifics for this class, namely just the jobHandler
+    return (Input,self._externalRun(Input,jobHandler))
```

Comment:
The evaluateSample docstrings have all been checked and will be updated in next push.